corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -103,13 +105,15 @@
 
 <script type="text/template" id="case-view-item-template">
     <% _.each(data, function(datum, index) { %>
-      <% if (styles[index].displayFormat === 'Image') { %>
-        <td class="module-caselist-column"><img class="module-icon" src="<%= resolveUri(datum) %>"/></td>
-      <% } else if (styles[index].displayFormat === 'Graph') { %>
-        <td class="module-caselist-column"><iframe srcdoc="<%= datum %>" height="300" width="300"></iframe></td>
-      <% } else { %>
-        <td class="module-caselist-column"><%= datum %></td>
-      <% } %>
+        <% if (!(styles[index].widthHint === "0")) { %>
```

Comment:
nit: `styles[index].widthHint !== "0"`

how come this one compares to the string `"0"` and the one above compares to the number `0`